packages/angular/cli/lib/config/workspace-schema.json

```diff
@@ -516,6 +516,28 @@
                 }
               }
             },
+            {
```

Comment:
You missed to add `@angular-devkit/build-angular:jest` in https://github.com/angular/angular-cli/blob/b492080059e792913485161779261fdaf5f3b0ea/packages/angular/cli/lib/config/workspace-schema.json#L363

Reply:
Ah, good catch. Updated.